askama_shared/src/generator.rs

```diff
@@ -1184,7 +1184,7 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
             return Ok(DisplayWrap::Unwrapped);
         }
 
-        if name == "escape" || name == "safe" || name == "e" || name == "json" {
+        if name == "escape" || name == "safe" || name == "e" || name == "json" || name == "yaml" {
```

Comment:
I think we should probably extract this list out into a constant or function or something, to avoid the repetition. Would you mind adding that in a separate commit?

Reply:
Just to be clear, you mean doing e.g. `FILTERS.contains(&name)`, right?

Reply:
Yup. Though this would be a newly defined `FILTERS`, right?